mqtt_client_interfaces/srv/NewMqttToRos2.srv

```diff
@@ -0,0 +1,11 @@
+# This service is used to dynamically register new MQTT -> ROS2 mappings. 
+# Syntax is the same as the ones used on config files.
+# Remember to also register ROS2 -> MQTT.

```

Comment:
In many of our own use cases, you only want to have uni-directional bridges on a single host, e.g., if sending ROS data via MQTT to another host.

```suggestion
# This service is used to dynamically register new MQTT -> ROS mappings.

```